ingestion/src/metadata/ingestion/api/topology_runner.py

```diff
@@ -182,17 +187,21 @@ def sink_request(self, stage: NodeStage, entity_request: C) -> Iterable[Entity]:
         """
 
         # Either use the received request or the acknowledged Entity
-        entity = entity_request
+        entity = entity_request.right
 
         if stage.nullable and entity is None:
```

Comment:
wouldn't this be `if not stage.nullable` if something can be nullable (ie `stage.nullable is True`) then we should not throw a ValueError right?